src/hotspot/share/opto/vector.cpp

```diff
@@ -515,41 +518,7 @@ void PhaseVector::expand_vunbox_node_vector(VectorUnboxNode* vec_unbox) {
     PhaseGVN& gvn = kit.gvn();
 
     Node* vec_val_load = get_loaded_payload(vec_unbox);
-
```

Comment:
Maybe directly merge `get_loaded_payload` method with this method? Any other place that `get_loaded_payload` is used besides here?